toy_problem/src/main.cpp

```diff
@@ -97,7 +100,7 @@ RobotState estimate(const RobotState& initial_state) const final
   {
     const auto& itinerary = result->get_itinerary();
     state.finish_time = rmf_traffic::time::to_seconds(
-        *itinerary.back().trajectory().finish_time() - start_time);
+        *itinerary.back().trajectory().finish_time() - std::chrono::steady_clock::now());
```

Comment:
This was previously computed as the duration of the task. Changing this to now() will make sure it is an absolute time to now. Certain times this will increase the number of expansion.

Reply:
Thanks for catching this!